saleor/graphql/product/mutations/products.py

```diff
@@ -341,11 +358,21 @@ def user_is_allowed(cls, user, input):
 
 
 class ProductTypeInput(graphene.InputObjectType):
-    name = graphene.String()
-    has_variants = graphene.Boolean(required=True)
-    product_attributes = graphene.List(graphene.ID)
-    variant_attributes = graphene.List(graphene.ID)
-    is_shipping_required = graphene.Boolean(required=True)
+    name = graphene.String(description='Name.')
+    has_variants = graphene.Boolean(
+        required=True,
+        description="""Determines if variants are allowed
+        for this product type.""")
+    product_attributes = graphene.List(
+        graphene.ID,
+        description='List of product type specific attributes.')
+    variant_attributes = graphene.List(
+        graphene.ID,
+        description='List of variant of this type specific attributes.')
```

Comment:
This doesn't sound natural, maybe something like this: "List of attributes used to distinguish between different variants of a product."